lambda-ci-resource.yaml

```diff
@@ -0,0 +1,32 @@
+include:
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/base-ci-resource.yaml

```

Comment:
I'm not sure you can do multi-level includes. I think the file which includes this file needs to just include both.

Should be able to test this by referring directly to the commit locked url for  this file

https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/b75241dfea814275e813a47b19715e6bb44d8e57/lambda-ci-resource.yaml

Reply:
I tested it with email-worker's ci lint checker and I saw the no op for publish, but maybe that's not completely correct?